lib/core/pmds/net/memif/memif_socket.c

```diff
@@ -986,15 +1004,18 @@ cne_memif_connect_client(struct cne_pktdev *dev)
     }
 
     sun.sun_family = AF_UNIX;
+    sunlen         = sizeof(struct sockaddr_un);
     if (pmd->flags & CNE_ETH_MEMIF_FLAG_SOCKET_ABSTRACT) {
         /* abstract address */
         sun.sun_path[0] = '\0';
         strlcpy(sun.sun_path + 1, pmd->socket_filename, CNE_MEMIF_SOCKET_UN_SIZE - 1);
+        sunlen = CNE_MIN(strlen(pmd->socket_filename) + 1, CNE_MEMIF_SOCKET_UN_SIZE) + sizeof(sun) -
+                 sizeof(sun.sun_path);
     } else {
         strlcpy(sun.sun_path, pmd->socket_filename, CNE_MEMIF_SOCKET_UN_SIZE);
```

Comment:
{} around single line statements is not required.

Reply:
Removed.